### PR TITLE
fix(MSA-419): update dates from local datetime to utc

### PR DESCRIPTION
**Pull request by behrindzhambaz**
dateTime's are saved with local time, they need to be saved in UTC

### CODE REVIEW

File: api/sql/schema/750-rule.decision.fetch.sql [hunk 1]
```diff
@@ -34,7 +34,7 @@ BEGIN
         countMonthly BIGINT
     )
 
-    SET @operationDate = ISNULL(@operationDate, GETDATE())
+    SET @operationDate = ISNULL(@operationDate, GETUTCDATE())
```
Comment (kalinkrustev) [inline api/sql/schema/750-rule.decision.fetch.sql:37]: This looks like a breaking change. If `@operationDate` parameter is in UTC, then `operationStartDate` and `operationEndDate` columns have to be in UTC too, which I am not sure is the case. Did you verify that?

Comment (BozhidarTodorov) [reply api/sql/schema/750-rule.decision.fetch.sql:37]: Probably could break current implementations. However, I think for sure we need all dates that are used for visualizing them in frontend to be UTC in database. We had huge amount of issues in previous old projects like ABZ where mobile apps cannot format to local datetime and we had to hardcode the timezone specifically for Zambia. Maybe we can do a major version in order to unblock ourselves as mobile developers are waiting the backend. @kalinkrustev , what is your opinion?

**Comment by kalinkrustev** [comment]
Please move the changes to a branch with name major/utc

**Comment by BozhidarTodorov** [comment]
Hello @kalinkrustev , new branch has been created:
https://github.com/softwaregroup-bg/ut-rule/commits/major/utc

Unfortunately I don't have an access to push to major branches and cannot merge latest master changes + do modifications if needed. However, I have checked and confirm that operationStartDate and operationEndDate as stored in UTC time in the database.:
https://github.com/softwaregroup-bg/ut-front-react/blob/master/components/DatePicker/Simple/index.js#L16

Date Picker stores its value in ISO String. 

We will checkout that branch in wallet project and do some test in order to see if everything works as expected. May I ask you to merge master into major/utc branch as you are the only person who can push to protected branches?

**Pull request closed without merging**